mongoengine/queryset/base.py

```diff
@@ -19,13 +18,19 @@
 from mongoengine.common import _import_class
 from mongoengine.connection import get_db
 from mongoengine.context_managers import switch_db
-from mongoengine.errors import (InvalidQueryError, LookUpError,
-                                NotUniqueError, OperationError)
+from mongoengine.errors import (
```

Comment:
@wojcikstefan I use package called `importanize` to organize imports. It did this :(

https://pypi.python.org/pypi/importanize